### PR TITLE
bt-546: [BUG] "Doesn't consider" is cut off on the talent preview page

### LINKED ISSUE #509: [BUG] Wrong displaying type of "Experience year" on the Preview page
**Description:**
Wrong displaying type of Experience year on the Preview page

**Preconditions:**
BSA talent app is open

**Environment:**
Android 13
Samsung Galaxy A52 (2340x1080)

**Steps to reproduce:**
1. Sign up with valid data
2. Fill in all steps
3. Go to the Preview page
4. Pay attention to the type of "Experience year"

**Actual Result:**
Experience year is displayed as 2.00 or 1.50

**Expected Result:**
Experience year is displayed as 2 or 1.5

**Visual Proof (screenshots, videos, text):**
image
image

**Severity:**
Minor (S4)

### LINKED ISSUE #546: [BUG] "Doesn't consider" is cut off on the talent preview page
**Description:**
"Doesn't consider" is cut off on the talent preview page

**Preconditions:**
The BSA app is open

**Environment:**
Android 13
Samsung Galaxy A52 (2340x1080)

**Steps to reproduce:**
1. Sign up with valid credentials
2. Fill out all steps
3. Go to the Preview page
4. Pay attention to the "Doesn't consider"

**Actual Result:**
"Doesn't consider" is cut off on the talent preview page

**Expected Result:**
All selected "Doesn't consider" options are fully visible on the talent preview page

**Visual Proof (screenshots, videos, text):**
Screenshot_4
Screenshot_5

**Severity:**
Minor (S4)

### PR DESCRIPTION
**Pull request by illia-dobro**
as

### CODE REVIEW

**Review by likeri29 — APPROVED** [review]

File: mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx [hunk 1]
```diff
@@ -37,6 +37,8 @@ const ProfilePreview: React.FC = () => {
         description,
     } = onboardingData;
 
+    const experienceYearsToDisplay = experienceYears && +experienceYears;
```
Comment (katerynakostikova) [inline mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx]: why do we need this?

Comment (illia-dobro) [reply mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx]: To display numbers like 1 or 0.5, the variable "experienceYears" is a string, although the editor shows it as a number. I'm not sure why.

File: mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx [hunk 2]
```diff
@@ -89,7 +91,7 @@ const ProfilePreview: React.FC = () => {
                             category={TextCategory.BODY1}
                             style={globalStyles.pl10}
                         >
-                            {experienceYears} year of experience
+                            {experienceYearsToDisplay} year of experience
```
Comment (katerynakostikova) [inline mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx]:
```suggestion
                            {+experienceYears} year of experience
```

File: mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx (hunk 1 above)
Comment (katerynakostikova) [inline mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx]:
```suggestion
```

File: mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx (hunk 2 above)
Comment (illia-dobro) [reply mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx]: experienceYears: number | null, so I have to add {experienceYears && +experienceYears} year of experience

**Review by katerynakostikova — APPROVED** [review]

**Pull request merged**